Tests/CryptoTests.swift

```diff
@@ -69,11 +77,21 @@ class CryptoTests: XCTestCase {
       return
     }
 
-    XCTAssertEqual(result.signature, [208, 47, 19, 208, 168, 253, 44, 130, 231, 240, 15, 213, 223, 59, 178, 60, 130, 146, 175, 120, 119, 21, 237, 130, 115, 88, 31, 213, 202, 126, 150, 205, 13, 237, 56, 251, 254, 240, 202, 228, 141, 180, 235, 175, 184, 189, 172, 121, 43, 25, 235, 97, 235, 140, 144, 168, 32, 75, 190, 101, 126, 99, 117, 13])
+    XCTAssertEqual(result.signature,
```

Comment:
For multi params that go over 100 (or 120 characters) it should look like this

XCTAssertEqual(
  result.signature,
  [208, 47, 19, 208, 168, 253, 44, 130, 231, 240, 15, 213, 223, 59, 178, 60, 130, 146, 175, 120, 119,
  21, 237, 130, 115, 88, 31, 213, 202, 126, 150, 205, 13, 237, 56, 251, 254, 240, 202, 228, 141, 180,
   235, 175, 184, 189, 172, 121, 43, 25, 235, 97, 235, 140, 144, 168, 32, 75, 190, 101, 126, 99, 117, 13])

Reply:
I'm not sure I understand because of the way Github formatted your comment? 

You want to line break after `Equal(` then +2 and align the params? I changed here to show what I think you mean, will update elsewhere when you've confirmed.

I don't see a rule to opt into to get this in the linter, LMK if you know of one.

Reply:
Yeah sorry I should have made it a code block like below.  Always newline after ( and +2 indent for lines that are > 100

Reply:
Done - fixed here and elsewhere. I pulled in a swift lint rule that enforces this automatically as well.